### DISCUSSION TITLE
AtlasPlaybook_v0.4.0部署后打开realtek audio console音频驱动面板闪退

### DISCUSSION THREAD

**Discussion by x51d6g4** [Atlas-OS/Atlas#1174] [Help & Support]
AtlasPlaybook_v0.4.0部署后打开realtek audio console音频驱动面板闪退，
没部署前可以打开，部署后打开闪退   
很需要realtek audio console
![Uploading 微信截图_20240803105017.png…]()

**Comment by he3als** [comment]
Can you specify your device model and where you're getting your drivers? Also, it seems like the screenshot wasn't uploaded.

#### 翻译版本：(Translated version)

您能具体说明一下您的设备型号以及驱动程序的来源吗？另外，似乎没有上传屏幕截图。

**Reply by x51d6g4** [reply]
CPU: AMD 5600X
主板：微星 MAG B550M MORTAR WIFI
内存：16GB
显卡：AMD RX6600XT
驱动全部来自微星官网
realtek audio console音频管理器是来自微软商店
realtek audio console音频管理器我是在部署前已经装好测试过的，但是部署后再打开realtek audio console音频管理器就闪退，像是强制关闭一样！

**Reply by x51d6g4** [reply]
如果需要截图，可能要晚一点才能给到，我现在在工作室

**Comment by x51d6g4** [comment]
这是屏幕录屏
https://github.com/user-attachments/assets/9a996f2d-26b8-4c57-bdfa-62fd3de68950

**Comment by 95271024** [comment]
系统啥版本 我之前装了个第三方精简版本的win,搞完后也打不开这个 只能重新装了系统

**Reply by the-P1neapple** [reply]
We only support official versions of Windows 10 or 11 operating systems. Please do not use AtlasOS on pre-modified installations.

我们仅支持 Windows 10 或 11 操作系统的正式版本。请不要在预先修改的安装上使用 AtlasOS。

**Comment by the-P1neapple** [comment]
@x51d6g4 
Please ensure you have the audio drivers for your motherboard installed in order to use the Realtek Audio Console. It will not work without them. You can download the drivers from the link provided.

请确保您已安装主板的音频驱动程序，以便使用 Realtek Audio Console。否则将无法使用。您可以从提供的链接下载驱动程序。

https://www.msi.com/Motherboard/MAG-B550M-MORTAR-WIFI/support#driver
